src/LeagueToolkit/LeagueToolkit.csproj

```diff
@@ -24,12 +24,21 @@
   <ItemGroup>
     <PackageReference Include="CommunityToolkit.Diagnostics" Version="8.0.0" />
     <PackageReference Include="CommunityToolkit.HighPerformance" Version="8.0.0" />
-    <PackageReference Include="FlatSharp" Version="6.2.1" />
+    <PackageReference Include="FlatSharp.Compiler" Version="7.0.2">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="FlatSharp.Runtime" Version="7.0.2" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.4.0" />
```

Comment:
What's the reason behind adding this package ?
I'm asking because it's being used in `MetaDump` without already being included as a package, which is quite strange.

Reply:
The previously used `FlatSharp` package has a dependency on `Microsoft.CodeAnalysis.CSharp.Workspaces` which in turn depends on `Microsoft.CodeAnalysis.CSharp`. The runtime package is much smaller and doesn't have those dependencies anymore.

Reply:
Ahh I didn't think of that, makes sense.